compiler/src/dotty/tools/dotc/transform/UncacheGivenAliases.scala

```diff
@@ -0,0 +1,67 @@
+package dotty.tools.dotc
+package transform
+
+import MegaPhase._
+import core.DenotTransformers.{IdentityDenotTransformer}
+import core.Symbols._
+import core.Contexts._
+import core.Types._
+import core.Flags._
+import core.StdNames.nme
+import core.Constants.Constant
+import core.Decorators._
+import core.TypeErasure.erasure
+import ast.tpd
+
+object UncacheGivenAliases:
+  val name: String = "uncacheAliasImplicits"
```

Comment:
Looks like this should be:
```suggestion
  val name: String = "uncacheGivenAliases"
```